care/emr/management/commands/load_emr_utils.py

```
@@ -0,0 +1,281 @@
+            facility=facility,
+            resource_type=resource_type,
+            resource_sub_type="other",
+            title=category_title,
+            slug=category_slug,
+            description=f"Auto-generated category for {category_title}",
+            created_by=created_by,
+            updated_by=created_by,
+        )
+        category.save()
+        logger.info("Created category: %s", category_title)
+        return category
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)